src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -1076,4 +1077,41 @@ class LicenceController(private val licenceService: LicenceService) {
   ) {
     licenceService.updateSentenceDates(licenceId, request)
   }
+
+  @GetMapping(value = ["/unapproved-by-crd"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @ResponseBody
+  @Operation(
+    summary = "Get details of edited licences which have not been reapproved by CRD date",
+    description = "Returns prisoner and com details " +
```

Comment:
description not quite right?

Reply:
moved into its own controller - UnapprovedLicenceController